packaging/aznfs/RPM/aznfs.spec

```diff
@@ -32,6 +36,59 @@ if [ "$init" != "systemd" ]; then
 	exit 1
 fi
 
+if grep -i "mariner" /etc/os-release > /dev/null 2>&1; then
```

Comment:
nit, this can be replaced with

if grep -qi "mariner" /etc/os-release; then

that lets errors show up on the console which might be useful